azure-functions/number-generator-function/package-lock.json

```diff
@@ -32,8 +31,7 @@
     },
     "node_modules/@azure/functions": {
       "version": "1.2.3",
-      "resolved": "https://registry.npmjs.org/@azure/functions/-/functions-1.2.3.tgz",
-      "integrity": "sha512-dZITbYPNg6ay6ngcCOjRUh1wDhlFITS0zIkqplyH5KfKEAVPooaoaye5mUFnR+WP9WdGRjlNXyl/y2tgWKHcRg=="
```

Comment:
Seems like we've lost a lot of `resolved` and `integrity' fields and _down_graded a lot of versions. Just wanted to check that this is expected